features/netsocket/cellular/generic_modem_driver/PPPCellularInterface.cpp

```diff
@@ -263,9 +263,11 @@ PPPCellularInterface::PPPCellularInterface(FileHandle *fh, bool debug)
     _fh = fh;
     _debug_trace_on = debug;
     _stack = DEFAULT_STACK;
+    _connection_status_cb = NULL;
+    _ppp_cb = Callback<void(nsapi_connection_status_t, int)>(this, &PPPCellularInterface::ppp_status_cb);
```

Comment:
I think you should be able to simplify this to `callback(this, &PPPCellularInterface::ppp_status_cb)`. Although not sure why it's a variable at all? Just pass that straight to the bringup function.